bots/behaviorLib.lua

```diff
@@ -909,6 +909,23 @@ function behaviorLib.MoveExecute(botBrain, vecDesiredPosition)
 	local vecMyPosition = unitSelf:GetPosition()
 	local vecMovePosition = vecDesiredPosition
 	
+	if unitSelf:GetTypeName() ~= "Hero_Fairy" then

```

Comment:
To not have to check this every frame, it'd possibly be better to have a var, that if nil (default) it fills it with true if it should run this. Then it simply checks if that is true before running.

(above)
local bIsntNymphora
(here)
if (bIsntNymphora == nil) then bIsntNymphora = unitSelf:GetTypeName() ~= "Hero_Fairy" end
if (bIsntNymphora) then ...